hummingbot/connector/derivative/coinflex_perpetual/coinflex_perpetual_order_book_tracker.py

```diff
@@ -0,0 +1,63 @@
+import asyncio
+import logging
+from collections import defaultdict, deque
+from typing import Deque, Dict, List, Optional
+
+import hummingbot.connector.derivative.coinflex_perpetual.constants as CONSTANTS
+from hummingbot.connector.derivative.coinflex_perpetual.coinflex_perpetual_api_order_book_data_source import (
+    CoinflexPerpetualAPIOrderBookDataSource,
+)
+from hummingbot.core.api_throttler.async_throttler import AsyncThrottler
+from hummingbot.core.data_type.order_book_message import OrderBookMessage
+from hummingbot.core.data_type.order_book_tracker import OrderBookTracker
+from hummingbot.core.utils.async_utils import safe_ensure_future
+from hummingbot.core.web_assistant.web_assistants_factory import WebAssistantsFactory
+from hummingbot.logger import HummingbotLogger
+
+
+class CoinflexPerpetualOrderBookTracker(OrderBookTracker):
```

Comment:
A new specific tracker class for the connector should be created only when the implementation of the tracker functionality should be modified. In this case the tracker does not overwrite any of the tracker logic, and it does not make sense to have a new tracker just to pass the `is_funding_info_initialized` to the data source. The class should be removed and the connector should use instead an instance of the generic tracker.